modules/nextflow/src/main/groovy/nextflow/Session.groovy

```diff
@@ -248,7 +254,9 @@ class Session implements ISession {
 
     private int poolSize
 
-    private List<TraceObserver> observers = Collections.emptyList()
+    private List<TraceObserver> observersV1 = Collections.emptyList()
+
+    private List<TraceObserverV2> observersV2 = Collections.emptyList()
```

Comment:
Is not ` Collections.emptyList()` immutable?

Reply:
Yes, but `obsserversV2` is reassigned in `init()`